src/Shout/Behavior.extension.st

```diff
@@ -0,0 +1,10 @@
+Extension { #name : #Behavior }
+
+{ #category : #'*Shout' }
+Behavior >> syntaxHighlighter [
```

Comment:
I don't think this should be in the PR, right?

Unless the PR is about pluggable highlighting, not about parse plugin mechanism.

Reply:
is about both. One is the proof/first use of the other.

Reply:
Then changing the PR title was enough.

I don't understand why the commit title makes me happy, personally, it does not make me happy.